router/router.go

```diff
@@ -365,7 +366,7 @@ func (r *Router) serveHTTP(server *http.Server, errChan chan error) error {
 		}
 	}
 
-	r.logger.Info("tcp-listener-started", zap.Object("address", r.listener.Addr()))
+	r.logger.Info("tcp-listener-started", slog.Any("address", r.listener.Addr()))
```

Comment:
I see the same difference here.

BEFORE
```
{
  "log_level": 1,
  "timestamp": "2024-09-09T17:25:48.705801789Z",
  "message": "tcp-listener-started",
  "source": "vcap.gorouter.router",
  "data": {
    "address": {
      "IP": "0.0.0.0",
      "Port": 80,
      "Zone": ""
    }
  }
}
```

AFTER
```
{
  "log_level": 1,
  "timestamp": "2024-09-09T20:40:24.065323310Z",
  "message": "tcp-listener-started",
  "source": "vcap.gorouter.router",
  "data": {
    "address": "0.0.0.0:80"
  }
}
```

Reply:
Please see comment above.